src/accounts/DeleteFreeAccountOverlay.tsx

```diff
@@ -20,28 +20,29 @@ import {
 } from '@influxdata/clockface'
 import {track} from 'rudder-sdk-js'
 
+// Context
+import {
+  DeleteFreeAccountContext,
+  VariableItems,
+} from 'src/accounts/context/DeleteFreeAccountContext'
+
+import {getOrg} from 'src/organizations/selectors'
```

Comment:
TODO: replace with identity selector

Reply:
Let's def change this to the identity selector.